build/NuSpecs/MUXControls-Nuget-FrameworkPackage.props

```
@@ -73,7 +73,7 @@
   </ItemGroup>
 
   <!-- Add a dependency for the app against VCLibs since we need it but the consuming app might be .NET only. -->
-  <ItemGroup>
+  <ItemGroup Condition="'$(ProjectExt)' != '.vcxproj' and ('$(PlatformTarget)' == 'x86' or '$(PlatformTarget)' == 'x64' or '$(PlatformTarget)' == 'ARM' or '$(PlatformTarget)' == 'ARM64')">
```

I think we still need the VCLibs 14 SDKReference since there might be a world where a project is .vcxproj but needs a different VCLibs version. Can we just remove that part and only do the PlatformTarget bit?

```suggestion
  <ItemGroup Condition="('$(PlatformTarget)' == 'x86' or '$(PlatformTarget)' == 'x64' or '$(PlatformTarget)' == 'ARM' or '$(PlatformTarget)' == 'ARM64')">
```

---

We could, but vcxproj calls the targets something different for x86 (Win32). I think the property might also be named something different.
Alternatively we could just make it `!='AnyCPU'`

---

Yeah, I was looking into that as we've hit that too. I think in that case the *Platform* variable is Win32 but the *PlatformTarget* is x86? In any case, if that happened then PlatformTarget would be Win32 and this check wouldn't fire and the SDKReference wouldn't get added. I think that's fine.

Or checking for != AnyCPU works for me too.